File: python/problem-2/DoubleOddNumbers2.py
def double_odds(numbers):
    result = []

    for number in numbers:
        if number % 2 == 1:
            double = number * 2
            result.append(double)
        else:
            result.append(number)
    
    return result


result = double_odds([1, 2, 3 , 4, 5, 6, 7])

print(result)

File: javascript/problem-2/DoubleOddNumbers2.js
function OddsNumbers(numbers) {
    const OddsNumbers = [];

    for (number of numbers) {
        if (number % 2 == 1) {
            const double = number * 2;
            OddsNumbers.push(double);
        } else {
            OddsNumbers.push(number);
        }
    }
    return OddsNumbers
}

result = OddsNumbers([1, 2, 3, 4, 5, 6, 7])

console.log(result)